app/components/ui/files/publish-modal/template.hbs

```diff
@@ -97,28 +99,37 @@
 </div>
 
 <div class="content">
-    <div class="ui form"></div>
+    <div class="ui form">
         <div class="grouped fields">
             <div class="field">
-                <div class="ui radio checkbox">
-                    <input type="radio" name="CC0" checked="checked">
-                    <label>Creative Commons Public Domain CCO <i class="info circle blue icon" ></i> </label> 
+                <div class="ui radio checkbox cc0" id='0'>
```

Comment:
Is this an Ember-y way to do this? Does Ember tend to grab values off the DOM when they're needed or does it use more of a two-way-data-bindy pattern? This is where my knowledge of Ember kinda breaks down but I wonder if this might instead be written more like:

```js
{{#each availableLicenses as |license|}}
  ...
  <input ... {{action "updateLicense" license}}
  ...
{{/each}}
```

So that the license is already set when it's needed.